### PR TITLE
Fix Libertalia provider

### PR DESCRIPTION
**Pull request by adaur**
Correct regex which makes Libertalia unusable

[Libertalia] :: Error while searching Libertalia, skipping: nothing to repeat

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: sickbeard/providers/libertalia.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def _doSearch(self, search_params, search_mode='eponly', epcount=0, age=0, epObj
                 with BS4Parser(data, features=["html5lib", "permissive"]) as html:
                     resultsTable = html.find("table", {"class" : "torrent_table"})
                     if resultsTable:
-                        rows = resultsTable.findAll("tr", {"class" : re.compile("torrent_row(.*)?")})
+                        rows = resultsTable.findAll("tr", {"class" : re.compile("torrent_row\.*")})
```
Comment (miigotu) [inline sickbeard/providers/libertalia.py]: the backslash should not be there. This is forcing only "new" torrents

Comment (adaur) [inline sickbeard/providers/libertalia.py]: What do you suggest then? The old solution was at least working

Comment (miigotu) [inline sickbeard/providers/libertalia.py]: `re.compile("torrent_row.*")` maybe, I dont have an account so I cant test it. The old solution didn't work for anything older than a few days.

**Comment by fernandog** [comment]
@duramato do you have an account? or @adaur can lend a user/pass or invite so we can test

**Comment by duramato** [comment]
@fernandog dont have an account there ,if someone has an invite im happy to test..

**Comment by maxcore** [comment]
i have invite give me email

**Comment by miigotu** [comment]
@duramato should get that account, he can lend it to me for testing if necessary.

**Comment by adaur** [comment]
Give me your email address. 

Otherwise, a torrent stays flagged as new until you open it... So it doesn't really matter, since 2 years old torrents will still be new for you as long as you never visited its page

**Comment by duramato** [comment]
my email is `goncalo.antonio.matos@gmail.com`
Dunno wich one of you will send the invite ,so leave a comment here wheb you do so that the other doesnt send another :P 
And for me to remove the email from the comment to avoid bots and crawlers :/

**Comment by adaur** [comment]
I can send it to both of you tonight.

**Comment by adaur** [comment]
Sorry, I forgot :(

@duramato sent
@miigotu give me your mail if you want one

Please try to keep your ratio > 1 and don't hit & run if you download files :D

**Comment by duramato** [comment]
@adaur didnt received it ,are you sure you sent it to the correct email?

**Comment by miigotu** [comment]
miigotu@gmail

**Pull request closed without merging**